lib/Instance.lua

```diff
@@ -155,7 +171,7 @@ end
 function Instance:IsA(className)
 	-- TODO: Hierarchy stuff
 
-	return self.ClassName == className
+	return self._internal.className == className
```

Comment:
This line should still work as it was written, right?

Reply:
It should, I think I was just annoyed at some other stuff not working. I'll change it.